projects/igniteui-angular/src/lib/tabs/tab-item.component.ts

```diff
@@ -163,4 +249,24 @@ export class IgxTabItemComponent implements IgxTabItemBase {
         const tab = tabsArray[index];
         tab.select(200);
     }
+
+    /**
+     * @hidden
+     */
+    public get template(): TemplateRef<any> {
+        if (this.relatedGroup && this.relatedGroup.customTabTemplate) {
+            return this.relatedGroup.customTabTemplate;
+        }
+        return this.defaultTabTemplate;
+    }
+
+    /**
+     * @hidden
+     */
+    public get context(): any {
+        if (this.relatedGroup) {
```

Comment:
This can be shorten to a single line using the ternary operator.

Reply:
Addressed.